### PR TITLE
70-find-listing-command

### LINKED ISSUE #70: Find Listings using keywords
Find Listings using keywords in Title

### PR DESCRIPTION
**Pull request by nt-nic**
## Which ticket does this address

- closes #70 

## What problems does this solve

- Unable to search for listings in the listing book according to keyword in the title

## Solution/Changes

- Added FindCommand, FindCommandParser and the relevant classes for testing.
- Added listings to the TypicalListing class to aid in testing

## Checklist

- [x] All tests pass locally and on CI

### CODE REVIEW

File: src/main/resources/view/MainWindow2.fxml [hunk 1]
```diff
@@ -13,48 +13,48 @@
...
-  <icons>
-    <Image url="@/images/GoodMatch_logo.png" />
-  </icons>
-  <scene>
-    <Scene>
-      <stylesheets>
-        <URL value="@DarkTheme.css" />
-        <URL value="@Extensions.css" />
-      </stylesheets>
+    <icons>
```
Comment (panpannnnn) [inline src/main/resources/view/MainWindow2.fxml:16]: will these tabs break anything?

**Review by panpannnnn — CHANGES_REQUESTED** [review]

File: src/main/resources/view/MainWindow2.fxml (hunk 1 above)
Comment (nt-nic) [reply src/main/resources/view/MainWindow2.fxml:16]: the tabs do not break anything, checkstyle and gradle clean build run correctly

Comment (nt-nic) [reply src/main/resources/view/MainWindow2.fxml:16]: The tabs were added due to intellij's auto indent for code style, they do not break anything.
Gradle clean build runs fine and the app is working normally

File: src/main/java/seedu/address/logic/commands/FindCommand.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.listing.TitleContainsKeywordsPredicate;
+
```
Comment (homuntan02) [inline src/main/java/seedu/address/logic/commands/FindCommand.java:8]: extra line here

File: src/main/java/seedu/address/logic/parser/FindCommandParser.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import java.util.Arrays;
+
+import seedu.address.logic.commands.FindCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.listing.TitleContainsKeywordsPredicate;
+
```
Comment (homuntan02) [inline src/main/java/seedu/address/logic/parser/FindCommandParser.java:10]: extra line here

**Review by homuntan02 — CHANGES_REQUESTED** [review]
Rest looks good!

File: src/main/java/seedu/address/logic/parser/FindCommandParser.java (hunk 3 above)
Comment (nt-nic) [reply src/main/java/seedu/address/logic/parser/FindCommandParser.java:10]: fixed

File: src/main/resources/view/MainWindow2.fxml (hunk 1 above)
Comment (nt-nic) [reply src/main/resources/view/MainWindow2.fxml]: fixed

File: src/main/java/seedu/address/logic/commands/FindCommand.java (hunk 2 above)
Comment (nt-nic) [reply src/main/java/seedu/address/logic/commands/FindCommand.java:8]: fixedd

**Review by homuntan02 — APPROVED** [review]
Looks good!!

**Pull request merged**